src/library/vicon_estimator.cpp

```diff
@@ -121,6 +143,45 @@ void TranslationalEstimator::setParameters(
   estimator_parameters_ = estimator_parameters;
 }
 
+//translational outlier rejection written by F.Grigis (fgrigis@student.ethz.ch) based on the code of Alex Millane
```

Comment:
Comment unnecessary. I believe this package has a contributors list in one file. If you wish, add your name there. Typically though this is not done.